toolchains/docker/toolchain.bzl

```diff
@@ -36,10 +37,12 @@ DockerToolchainInfo = provider(
 def _docker_toolchain_impl(ctx):
     toolchain_info = platform_common.ToolchainInfo(
         info = DockerToolchainInfo(
+            docker_flags = ctx.attr.docker_flags,
             client_config = ctx.attr.client_config,
             gzip_path = ctx.attr.gzip_path,
             gzip_target = ctx.attr.gzip_target,
             tool_path = ctx.attr.tool_path,
+            client_config = ctx.attr.client_config,
```

Comment:
this is duplicated from line 41

Reply:
> this is duplicated from line 41

derp.  I was having some merge issues.  Turns out my buildifier command was screwing up files.  I was using `buildifier --lint=fix --warnings=all -r ./` and it did some weird things.  What does your local buildifier command look like?